### PR TITLE
[15.0][FW] web_m2x_options: multiple ports from 12.0, 13.0, 14.0

### PR DESCRIPTION
**Pull request by nguyenhk99**

### CODE REVIEW

File: web_m2x_options/models/ir_config_parameter.py [hunk 1]
```diff
@@ -12,5 +12,9 @@ def get_web_m2x_options(self):
             "web_m2x_options.limit",
             "web_m2x_options.search_more",
             "web_m2x_options.m2o_dialog",
+            "web_m2x_options.field_limit_entries",
         ]
-        return self.sudo().search_read([["key", "in", opts]], ["key", "value"])
+        return {
+            res["key"]: res["value"]
+            for res in self.sudo().search_read([["key", "in", opts]], ["key", "value"])
+        }
```
Comment (hailangvn) [inline web_m2x_options/models/ir_config_parameter.py]: Just a non-blocking observation.
```suggestion
        values = self.sudo().search_read([["key", "in", opts]], ["key", "value"])
        return { res["key"]: res["value"] for res in values }
```

**Review by hailangvn — COMMENTED** [review]
LGTM

**Review by hailangvn — COMMENTED** [review]
LGTM

**Review by njeudy — APPROVED** [review]
Technical and functionnal review: Ok for me

**Pull request merged**